File: BubbleSort.java
import java.util.Arrays;

class Main {

  static void bubbleSort(int array[]) {
    int size = array.length;

    for (int i = 0; i < size - 1; i++)

      for (int j = 0; j < size - i - 1; j++)

        if (array[j] > array[j + 1]) {
          int temp = array[j];
          array[j] = array[j + 1];
          array[j + 1] = temp;
        }
  }

  public static void main(String args[]) {
      
    int[] data = { -2, 45, 0, 11, -9 };
  
    Main.bubbleSort(data);
    
    System.out.println("Sorted Array in Ascending Order:");
    System.out.println(Arrays.toString(data));
  }
}

File: Sorting/bubble_sort.py
def bubble_sort(array):
    size = len(array)
    
    for i in range(size - 1):
        swapped = False
        
        for j in range(size - i - 1):
            if array[j] > array[j + 1]:
                array[j], array[j + 1] = array[j + 1], array[j]
                swapped = True
        
        if not swapped:
            break


data = list(map(int, input("Enter numbers separated by spaces: ").split()))
bubble_sort(data)

print("Sorted Array in Ascending Order is:")
print(data)
